lib/src/kbasesearchengine/tools/SearchTools.java

```diff
@@ -25,6 +25,9 @@
 import java.util.UUID;
 
 import kbasesearchengine.common.FileUtil;
+import kbasesearchengine.events.StatusEventProcessingState;
```

Comment:
Removed. Since I don't have all the jars (manually) included some imports although used appear grayed out and blend with the ones that are not used. At some point if we move to something modern like gradle, this won't be a problem, but I'll remember to double check manually until then.